c2cpg/src/main/scala/io/shiftleft/c2cpg/astcreation/AstCreatorHelper.scala

```diff
@@ -235,15 +236,29 @@ trait AstCreatorHelper {
     }
   }
 
+  private def astforDecltypeSpecifier(decl: ICPPASTDecltypeSpecifier, order: Int): Ast = {
+    val op = "operators.<typeOf>"
```

Comment:
Is this operator defined in the schema, and if so, can we reference as constant?

Reply:
Sadly, its not. We might want to add it as its quite common.